spec/features/idv/verify_by_mail_pending_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'a user that is pending verify by mail', allowed_extra_analytics: [:*] do
+  include IdvStepHelper
+
+  it 'requires them to enter code or cancel to enter the proofing flow' do
+    user = create(:user, :fully_registered)
+    profile = create(:profile, :with_pii, :verify_by_mail_pending, user: user)
+    create(:gpo_confirmation_code, profile: profile, created_at: 2.days.ago, updated_at: 2.days.ago)
+
+    start_idv_from_sp(biometric_comparison_required: false)
+    sign_in_live_with_2fa(user)
+
+    expect(current_path).to eq(idv_verify_by_mail_enter_code_path)
+
+    # Attempting to start IdV should require enter-code to be entered
+    visit idv_welcome_path
+    expect(current_path).to eq(idv_verify_by_mail_enter_code_path)
+
+    # Cancelling redirects to IdV flow start
+    click_on t('idv.gpo.address_accordion.cta_link')
```

Comment:
This is wholly irrelevant to your PR, but just looking at this translation string I would never guess it read "Clear your information and start over". I would expect a CTA to be enticing the user to start some action, not cancel and start over.